stl/inc/__msvc_formatter.hpp

```diff
@@ -90,6 +90,9 @@ enum class _Basic_format_arg_type : uint8_t {
 static_assert(static_cast<int>(_Basic_format_arg_type::_Custom_type) < 16, "must fit in 4-bit bitfield");
 
 #if _HAS_CXX23
+_EXPORT_STD template <class _Ty>
+constexpr bool enable_nonlocking_formatter_optimization = false;

```

Comment:
Hold off on this for now, I have concerns about the Standard's wording. [format.formatter.spec]/3 says:
> Unless specified otherwise, for each type `T` for which a `formatter` specialization is provided by the library, each of the > headers provides the following specialization:
>   `template<> inline constexpr bool enable_nonlocking_formatter_optimization<T> = true;`

When I made this comment I was assuming that "the library" means "The Standard Library" (everything) but it might feasibly just mean "the `<format>` library". Many of the names I've listed above are actually names of _templates_, not types, which are not covered by the wording above. It doesn't make sense that we'd want the nonlocking formatter optimization for e.g. `char` but not `basic_string_view`, or `stacktrace_entry` but not `basic_stacktrace`. I've sent email to LWG asking for clarification, which will most likely result in an LWG issue once we've determined the intended design.

Why LWG decided it was a good idea to add this weird blanket wording instead of simply putting actual `enable_nonlocking_formatter_optimization` specializations in the header synopses is beyond me.